modules/metering-install-operator.adoc

```diff
@@ -22,4 +22,4 @@ Metering Operator.
 
 . On the *Installed Operators* screen the *Status* displays *InstallSucceeded* when metering has finished installing. Click the name of the operator in the first column to view the *Operator Details* page.
 
-From the *Operator Details* page you can create different resources related to metering. To complete your installation create a MeteringConfig resource to configure metering and install the components of the metering stack.
```

Comment:
Maybe a note after Line 23 that it might take a moment for the `Metering` operator to appear? I am using a 4.1 system and it took several seconds to show up, enough to make me think I screwed something up.